LSM303AGR.h

```diff
@@ -131,6 +150,22 @@ int32_t lsm303_acc_startFIFO(const ACC_FULL_SCALE_t RANGE, const ACC_OPMODE_t MO
  */
 int32_t lsm303_acc_stop();
 
+/** @brief Set the type, threhold value and time for Interrupt 2
+ * 
+ *  This function states exactly the interrupt mode, threshold and minimum duration
+ *  to activate interrupt 2 and set up the device properly.
+ * @return true if successful, system error code otherwise
+ */
+int32_t lsm303_acc_setINT2(ACC_INT2_type_t mode, uint8_t threshold, uint8_t duration);
```

Comment:
the comment needs to list the parameters, what they are, what the acceptable ranges are, what they represent, etc.
As written I have no idea what duration is, nor what unit it is in. Can I really pass in 0-255? 255 seconds, milliseconds? What is the threshold in? Gs? milligravities? can the threshold really be 255 gravities?

Reply:
if there are really only two modes why is it not a Boolean?